src/cli/cli.cpp

```diff
@@ -5464,6 +5464,13 @@ template <> otError Interpreter::Process<Cmd("ping")>(Arg aArgs[])
 
 #endif // OPENTHREAD_CONFIG_PING_SENDER_ENABLE
 
+template <> otError Interpreter::Process<Cmd("platform")>(Arg aArgs[])
+{
+    OT_UNUSED_VARIABLE(aArgs);
```

Comment:
May be good to verify if aArgs is empty.

Reply:
I saw lots of commands doesn't check it. Maybe here we don't want to enforce args being empty to leave room for future enhancement, such that we can add args in the future w/o breaking previous version.